src/lib.rs

```diff
@@ -35,3 +35,7 @@ pub use reader::Reader;
 pub use symbol_table::SymbolTable;
 pub use system_event_handler::SystemEventHandler;
 pub use types::IonType;
+
+pub mod reexport {
```

Comment:
I would just spell this a `ext` or if we really want to be verbose `external`.  Mentioned that spelling in #302, but I am not going to veto if we prefer `reexport`.

Reply:
Yeah, I left some thoughts in that issue :)

I'm not in ❤️ with `reexport`, but `ext` is more confusing.

Reply:
per slack, going with external. we agree with top-level..